src/app/participant-list/participant-list.component.html

```diff
@@ -222,9 +222,8 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
         </app-search-bar>
 
         <table class="table table-striped Table--Hover Table--Foot Width--100">
-          <!--#mf="mfDataTable" [mfRowsOnPage]="hasRole().getUserSetting().getRowsPerPage()"-->
           <thead>
-          <th></th> <!-- needed for the first column with checkbox -->
+          <th *ngIf="assignees != null && assignees.length > 0"></th> <!-- needed for the first column with checkbox -->
```

Comment:
I would go for this `*ngIf=Array.isArray(assigness) && assignees.length` more cleaner than ugly null check.